src/features/Home/GetStarted/GetStarted.module.scss

```diff
@@ -0,0 +1,82 @@
+.mainPageRow {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: fit-content;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  padding: 80px 0;
```

Comment:
You can now import `@use 'src/styles/utility' as *;` at the top of the css file and convert the pixel values to rem values. using the rem function. so in this padding case it would be for example: `padding: rem(8) 0;`. Please convert the px values to rem.